src/main/anovos/data_report/report_preprocessing.py

```diff
@@ -269,18 +267,33 @@ def charts_to_objects(spark, idf, list_of_cols='all', drop_cols=[], label_col=No
 
     num_cols, cat_cols, other_cols = attributeType_segregation(idf.select(list_of_cols))
 
-    idf_cleaned = outlier_categories(spark, idf, list_of_cols=cat_cols, coverage=coverage, max_category=bin_size)
+    if cat_cols:
+        idf_cleaned = outlier_categories(spark, idf, list_of_cols=cat_cols, coverage=coverage, max_category=bin_size)
+    else:
+        idf_cleaned = idf
 
     if drift_detector:
         encoding_model_exists = True
+        binned_cols = spark.read.parquet(source_path + "/drift_statistics/attribute_binning")\
+                        .select('attribute').rdd.flatMap(lambda x:x).collect()
+        to_be_binned = [e for e in num_cols if e not in binned_cols]
     else:
         encoding_model_exists = False
-    idf_encoded = attribute_binning(spark, idf_cleaned, list_of_cols=num_cols, method_type=bin_method,
+        binned_cols = []
+        to_be_binned = num_cols
+    idf_encoded = attribute_binning(spark, idf_cleaned, list_of_cols=to_be_binned, method_type=bin_method,
+                                    bin_size=bin_size,
+                                    bin_dtype="categorical", pre_existing_model=False,
+                                    model_path=source_path + "/charts_to_objects", output_mode='append')
```

Comment:
if 
to_be_beinned = num_cols and 
list_of_cols = to_be_binned
then is bin_dtype = "categorical" or should it be "numerical"